gamemodes/terrortown/gamemode/client/cl_target_data.lua

```diff
@@ -6,6 +6,52 @@ local IsColor = IsColor
 
 TARGET_DATA = {}
 
+---
+-- Initializes the @{TARGET_DATA} object
+-- @param entity ent The focused Entity
+-- @param entity unchangedEnt The original focused Entity if focus was changed by the hook
+-- @param number distance The distance to the focused Entity
+-- @return @{TARGET_DATA} The object to be used in the hook
+-- @internal
+-- @realm client
+function TARGET_DATA:Initialize(ent, unchangedEnt, distance)
```

Comment:
I would like to suggest renaming `unchangedEnt` to `originalEnt` to avoid confusion.

Reply:
**Name Change:**
I can do this, otherwise this name is used in the complete target_data struct. And well if it is named "originalEnt" we should also assign it this value. Right now it only gets assigned if there is a changed. Therefore the unchangedEnt is normally nil and only is the originalEnt if the entity got modified. So leaving this open for discussion. Maybe Tim has an opinion about this.
Normally I'd agree but then we also should change the codebase and that means, that the command GetUnchangedEntity could lead to incompatibilities.

**Note:**
Concerning the note, while this could make sense to write to every function I normally would assume of a library to be properly initialized, which is done just at a later point by TTT2.
So I wrote a note at the Initialization: 
https://github.com/TTT-2/TTT2/pull/669/files#diff-b3cb386207438f4819b734529e8233c5dacd3ce6a3f11b34d724ee9f62a6aaeeR44-R46
and it gets initialized directly at the file start of cl_targetid:
https://github.com/TTT-2/TTT2/pull/669/files#diff-8500b5865660f13e615fbd1ed93e16a62af9d1e3a6402dc34ed7786a2df8c8f1R17-R19

Reply:
I agree with @ZenBre4ker. Also I don't see a reason for a namechange to a variable that exists since half a year.